deploy/example/storageclass-blob-nfs.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 provisioner: blob.csi.azure.com
 parameters:
   protocol: nfs
+  useSASToken: true
```

Comment:
it should be `"true"`, otherwise it would return parsing error.
also don't change the default storage class setting in example.

Reply:
use following sed command example to change the storage class setting in  https://github.com/kubernetes-sigs/blob-csi-driver/blob/ec7f3822633e584b9bfa3b55757219c9a848f9d3/test/external-e2e/run.sh#L34-L36
```
sed -i 's/protocol: nfs/protocol: nfs\n  useSASToken: "true"/g' storageclass-blob-nfs.yaml
```

Reply:
done